client/src/pages/ManageBookings/ManageBookings.tsx

```diff
@@ -0,0 +1,155 @@
+import { useState, useEffect } from 'react';
+import Booking from '../../components/Booking/Booking';
+import { BookingRequest } from '../../interface/BookingRequest';
+import { mockBookingRequest } from '../../mocks/mockBookingRequest';
+
+import { Grid, Paper, Typography, Avatar, Container, Box } from '@material-ui/core';
+import { MuiPickersUtilsProvider, DatePicker } from '@material-ui/pickers';
+import DateFnsUtils from '@date-io/date-fns';
+import { MaterialUiPickersDate } from '@material-ui/pickers/typings/date';
+import CircularProgress from '@material-ui/core/CircularProgress';
+
+import { useAuth } from '../../context/useAuthContext';
+import { useSocket } from '../../context/useSocketContext';
+import { useHistory } from 'react-router-dom';
+
+import useStyles from './useStyles';
+
+const ManageBookings = (): JSX.Element => {
+  const { loggedInUser } = useAuth();
+  const { initSocket } = useSocket();
+
+  const history = useHistory();
+  const classes = useStyles();
+
+  useEffect(() => {
+    initSocket();
+  }, [initSocket]);
+
+  const [date, changeDate] = useState<MaterialUiPickersDate>(new Date());
```

Comment:
I think unless we have a good reason, we should resist the temptation to change the `useState` conventions 🤔  I agree `changeDate` makes just as much as `setDate`, but when we see `setDate` somewhere within the code we know it's most likely the setter from the `useState` call. 

I find one common use case for modifying setters coming from React state is when we abstract away state logic into a custom hook 👍🏼